addons/website_slides/views/website_slides_templates_homepage.xml

```diff
@@ -282,14 +282,17 @@
 </template>
 
 <template id='course_card' name="Course Card">
-    <div class="card w-100 o_wslides_course_card mb-4">
+    <div class="card w-100 o_wslides_course_card mb-4" t-attf-style="#{'opacity: 0.5;' if not channel.is_published else ''}">
```

Comment:
You should use a class. We avoid inline style because it breaks with themes, custom css, ...